halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/deck/DeckProfileFactory.java

```diff
@@ -72,7 +74,11 @@ protected void setProfile(
       Profile profile,
       DeploymentConfiguration deploymentConfiguration,
       SpinnakerRuntimeSettings endpoints) {
-    StringResource configTemplate = new StringResource(profile.getBaseContents());
+    ObjectResource configTemplate =
+        new ObjectResource(
+            profile
+                .getBaseContents()
+                .replace("version: version,", "version: version,\n  plugins: '{{%plugins%}}',"));
```

Comment:
this is hacky, but since we don't control the settings.js format, this works for now